src/plugins/webpush.js

```diff
@@ -11,7 +11,22 @@ class WebPush {
 	constructor() {
 		const vapidPath = path.join(Helper.getHomePath(), "vapid.json");
 
-		if (fs.existsSync(vapidPath)) {
+		let vapidStat = undefined;
+
+		try {
+			vapidStat = fs.statSync(vapidPath);
+		} catch {
+			// ignored on purpose, node v14.17.0 will give us {throwIfNoEntry: false}
+		}
+
+		if (vapidStat) {
+			if (Helper.isWorldReadable(vapidStat)) {
+				log.warn(
+					vapidPath,
+					"is world readable. The file contains secrets. Please fix the permissions"
```

Comment:
can we provide a link to directions to fix the instructions or include them here?

Reply:
I also think `isWorldReadable` should just live here unless you see another use for it in the nearish future.

Reply:
re inlining, sure will do.
In regards to how to fix it, well... Ignoring windows? No clue how to do it there but for unixy systems chmod will do.

Reply:
done. PTAL